packages/gatsby/src/schema/node-model.js

```diff
@@ -847,6 +847,7 @@ async function resolveRecursive(
   return _.pickBy(resolvedFields, (value, key) => queryFields[key])
 }
 
+const withResolverContext = require(`./context`).default
```

Comment:
Rogue `.default` tryin to sneak in here

Reply:
hmm yeah — I'll move it to an import at the top